.vscode/settings.json

```diff
@@ -7,6 +7,5 @@
     {"directory": "packages/hub", "changeProcessCWD": true},
     {"directory": "packages/jest-gas-reporter", "changeProcessCWD": true},
     {"directory": "packages/nitro-protocol", "changeProcessCWD": true}
-  ],
-  "editor.formatOnSave": true
```

Comment:
You can override this option in the wallet `settings.json`, so that it remains for the other folders.

I think this problem is temporary due to
- eslint not running prettier
- vscode still using prettier to format on save in the wallet, but using the wrong prettier config (?)